distribution/org.wso2.developerstudio.eclipse.distribution.project/src/org/wso2/developerstudio/eclipse/distribution/project/editor/DistProjectEditorPage.java

```diff
@@ -483,7 +483,10 @@ private void handleTreeItemEdit(final TreeItem item){
         if (lastCtrl != null){
         	lastCtrl.dispose();  
         }
-        if(nodeData.hasChildren() || !item.getChecked()) return;
+
+		// Fixing TOOLS-2837
+		// if(nodeData.hasChildren() || !item.getChecked())return;

```

Comment:
Please remove commented out code. If this was to fix a bug, no need to add a comment saying this is done to fix a bug and comment the code.